teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def intialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly do the same change for the parameters of the initialize constructor and pass it to super as in the previous comment